src/plugins/interactive-media/interactive-media-editor.js

```diff
@@ -42,13 +44,14 @@ function InteractiveMediaEditor({ content, onContentChanged }) {
   const clientConfig = useService(ClientConfig);
   const { formatPercentage } = useNumberFormat();
   const { t } = useTranslation('interactiveMedia');
-  const [sourceDuration, setSourceDuration] = useState(0);
   const interactiveMediaInfo = useService(InteractiveMediaInfo);
   const [selectedChapterIndex, setSelectedChapterIndex] = useState(0);
   const [selectedChapterFraction, setSelectedChapterFraction] = useState(0);
-  const [isDeterminingDuration, setIsDeterminingDuration] = useState(false);
   const { sourceType, sourceUrl, playbackRange, chapters, width, showVideo } = content;
 
+  const [mediaDuration] = useMediaDurations([urlUtils.getMediaUrl({ cdnRootUrl: clientConfig.cdnRootUrl, sourceType, sourceUrl })]);
+  const sourceDuration = mediaDuration.duration;
```

Comment:
`mediaDuration.duration` could be `null` (if it is still in the progress of being determined), so we cannot use it unguarded (to achieve the former functionality, `|| 0` should do the trick).

Reply:
Actually I had checked that and it cannot be `null`, it return the default "unresolved entry", the same object as for resolved, with default values and `isResolved` set to `false`

Reply:
Ah sorry, `duration` could be `null` you said, not the `mediaDuration`.
Honestly I would always default numbers to `0` unless they are IDs, like we said for the recent streamlining. Unless we saw that part differently actually.
What do you think?

Reply:
For me that would be fine, too. We could change it that way.

Reply:
Thanks, changed that way!

Reply:
:+1: